Lib/axisregistry/__init__.py

```diff
@@ -53,6 +53,32 @@
 )
 
 
+    for axis in [
+        "casual.textproto",
+        "cursive.textproto",
+        "fill.textproto",
+        "flair.textproto",
+        "grade.textproto",
+        "italic.textproto",
+        "monospace.textproto",
+        "optical_size.textproto",
+        "slant.textproto",
+        "softness.textproto",
+        "volume.textproto",
+        "weight.textproto",
+        "width.textproto",
+        "wonky.textproto",
+        "x_opaque.textproto",
+        "x_transparent_figures.textproto",
+        "x_transparent.textproto",
+        "y_opaque.textproto",
+        "y_transparent_ascender.textproto",
+        "y_transparent_descender.textproto",
+        "y_transparent_figures.textproto",
+        "y_transparent_lowercase.textproto",
+        "y_transparent_uppercase.textproto",
+    ]:
+        append_AxisMessage(resource_filename("axisregistry", "data/" + axis))
```

Comment:
It looks like we may need to add the following function that was removed in https://github.com/google/fonts/pull/5072 back?

```python
def append_AxisMessage(path):
    axis = get_Protobuf_Message(AxisProto, path)
    registry[axis.tag] = axis  # pylint: disable=E1101
```

Reply:
and 

```python
def get_Protobuf_Message(klass, path):
        message = klass()
        with open(path, "rb") as text_data:
            text_format.Merge(text_data.read(), message)
        return message
```

?